highlight_view.py

```diff
@@ -692,12 +692,27 @@ def open_tooltip(view, point, line_report=False):
     if not errors:
         return
 
+    def on_navigate(href: str) -> None:
+        if href == "copy":
+            sublime.set_clipboard(join_msgs_raw(errors))
+            sublime.active_window().status_message("SublimeLinter: info copied to clipboard")
+            view.hide_popup()
+
     tooltip_message = join_msgs(errors, show_count=line_report, width=80)
     view.show_popup(
-        TOOLTIP_TEMPLATE.format(stylesheet=TOOLTIP_STYLES, content=tooltip_message),
+        TOOLTIP_TEMPLATE.format(stylesheet=TOOLTIP_STYLES, content=tooltip_message) + "<a href=\"copy\">Copy</a>",
```

Comment:
Please add the link to the `TOOLTIP_TEMPLATE` in stead of just slapping it onto the end. Now the `<a>` sits outside the `<body>`, which is probably the cause of it being misaligned.

Reply:
@braver 
Just did this, thanks for the suggestion!

The copy link now looks like this. It's properly aligned horizontally with the rest of the content, and I gave it a bit of `margin-top` so it's not so vertically close.

<img width="525" alt="Screen Shot 2019-07-09 at 10 01 12 AM" src="https://user-images.githubusercontent.com/1524088/60899763-a9a7ff00-a230-11e9-8b34-f64d2ef03b98.png">